Documentation/environment-variables.rst

```diff
@@ -7,28 +7,27 @@ Communication Thread
 --------------------
 
 By default, any SeisSol run with more than one MPI rank will use a communication thread to advance the MPI progress engine.
-For that, you will need to leave at least one thread vacant in your OpenMP thread placing map, cf. the SuperMUC-NG example below.
+For that, you must leave at least one thread vacant in your OpenMP thread placing map, cf. the SuperMUC-NG example below.
```

Comment:
Maybe a bit finnicky to remark, but "must" feels a bit out of place here (since it's not an "inner" thing).

... how about "For that, it is required that you leave [...]" ?

(I agree that the `will need to` formulation etc. are a bit more informal)

Reply:
"It is required to leave"...